modules/ui/org.eclipse.fx.ui.preferences/src/main/java/org/eclipse/fx/ui/preferences/page/FloatFieldEditor.java

```diff
@@ -25,35 +27,44 @@
  * A Field editor for float preferences.
  * </p>
  */
-public class FloatFieldEditor extends FieldEditor {
-	private final TextField textField;
-
+public class FloatFieldEditor extends FieldEditor<Float> {
 	private static final Logger LOGGER = LoggerCreator.createLogger(FloatFieldEditor.class);
+	
+	private final TextField textField;
+	private SimpleFloatProperty value = new SimpleFloatProperty(this, "value");
 
 	public FloatFieldEditor(String name, String label) {
 		super(name, label);
 		this.textField = new TextField();
 		getChildren().add(textField);
 
-		configureValidation(this.textField);
+		NodeDecorator.apply(this.textField, statusProperty());
+		
+		this.textField.textProperty().addListener((obs, old, newValue) -> {
```

Comment:
Not a big thing (and applies to other editors as well) - if you are only interested in the newValue you could use `FXObservableUtil.onChange(Consumer)`

Reply:
:+1: Always typing all 3 change listener parameters was annoying :)